nix/default.nix

```diff
@@ -274,18 +317,48 @@ let
           stdio
           uri
           uutf
-          zarith # cmdliner easy-format js_of_ocaml ocp-ocamlres tls
+          zarith
+          cmdliner
+          # easy-format js_of_ocaml ocp-ocamlres tls
           # alcotest dum pprint
+          ocp-ocamlres
+          pprint
           ocamlgraph
           findlib
           genspio
         ] ++ [ libusb1 libusb1.out (gmp.override { withStatic = true; }) ];
         doCheck = false;
-        buildPhase = "dune build src/bin_client/tezos-client.install";
+        buildPhase = ''
+          # tezos-node build requires ocp-ocamlres binary in PATH
+          PATH=$PATH:${ocp-ocamlres}/lib/ocaml/4.07.1/bin
```

Comment:
should this just be in nativeBuildInputs?

Reply:
I think, that I've tried, and this hasn't worked.
Aha, it fails with 
```
File "src/lib_protocol_compiler/dune", line 4, characters 9-26:
4 |   (run %{bin:ocp-ocamlres} -format ocaml -o %{targets}
             ^^^^^^^^^^^^^^^^^
Error: Program ocp-ocamlres not found in the tree or in PATH
 (context: default)
```